FindSplash.cmake

```diff
@@ -70,6 +70,12 @@ set(Splash_FOUND TRUE)
 
 # find libSplash installation #################################################
 #
+
+# If environment variable is not set, but CMake var is, use the CMake var
```

Comment:
Note: env/var hints are [deprecated](http://public.kitware.com/pipermail/cmake-developers/2014-June/022421.html) in CMake for the sake of the common `CMAKE_PREFIX_PATH` that is understood by all relevant functions (`find_path`, `find_library`, etc.).